perk-store-services/src/main/resources/locale/addon/PerkStore_en.properties

```diff
@@ -71,7 +71,7 @@ exoplatform.perkstore.warning.recipientDoesntHaveWallet=The recipient doesn't ha
 exoplatform.perkstore.label.quantity=Quantity
 exoplatform.perkstore.label.yourOrders=Your orders
 exoplatform.perkstore.label.availableQuantity=Available quantity
-exoplatform.perkstore.label.quantityWithMax=Quantity (max: {0})
+exoplatform.perkstore.label.quantityWithMax=max: {0}
```

Comment:
Proposed to be changed on crowdin, see https://crowdin.com/translate/exo-platform/35598/en-en#6910668 @ArfaouiMohamedAmine (changing an existing translation should be done on crowdin)

Reply:
@boubaker @ArfaouiMohamedAmine  approved on crowdin.